.github/workflows/mypy_primer.yml

```diff
@@ -15,6 +15,9 @@ on:
     - 'mypy/test/**'
     - 'test-data/**'
 
+permissions:
```

Comment:
It duplicates these lines https://github.com/python/mypy/pull/13645/files#diff-1d3e91c57336ce4ff8a9d5762ddc5ffa66f99f0afb918aab0b02e53db80bf54cR25

Reply:
Hello @sobolevn, I analyzed all of the workflows included in the project and found that they only need contents:read to function hence I added it on all workflow files.

Reply:
We don't need these lines anymore: https://github.com/python/mypy/pull/13645/files#diff-1d3e91c57336ce4ff8a9d5762ddc5ffa66f99f0afb918aab0b02e53db80bf54cR25-R26

Reply:
Maybe there are other places where we already have `permissions:` set?

Reply:
@sobolevn yes, [mypy_primer_comment.yml](https://github.com/python/mypy/blob/master/.github/workflows/mypy_primer_comment.yml#L11) already has token permission set hence I did not include it in the PR. This change will increase the security of other workflow files.